CHANGES.md

```diff
@@ -1,3 +1,47 @@
+## DC/OS 1.10.7
+
+```
+* For any significant improvement to DC/OS add an entry to Fixed and Improved section.
+* For Security updates, please call out in Security updates section.
+* Add to the top of the existing list.
+* External Projects like Mesos and Marathon shall provide a link to their published changelogs.
+
+Format of the entries must be.
+
+* Entry with no-newlines. (DCOS_OSS_JIRA)
+<new-line>
+* Entry two with no-newlines. (DCOS_OSS_JIRA_2)
+```
+
+
+### Notable changes
+
+* Includes Marathon version [1.5.8](https://github.com/dcos/dcos/pull/2707)  (DCOS_OSS-2352)
+
+* Includes Mesos Version 1.4 [dcos-mesos-1.4.x-nightly-315d047](https://github.com/mesosphere/mesos/tree/dcos-mesos-1.4.x-nightly-315d047)
+
+* Includes Metronome version 0.4.2
+
+
+### Fixed and improved
+
+* Increase the mesos executor registration timeout from 2 seconds to 20 seconds. (DCOS_OSS-2335)
+
+* Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file. (DCOS-14199)
+
+* Block DC/OS install if mesos work dir or docker root dir is XFS but not mounted with ftype=1 (COPS-3158)
+
+* DCOS-UI: Update the version to [1.10.6](https://github.com/dcos/dcos/pull/2788)
+
+
+### Security Updates
+
+* Disable the 3DES bulk encryption algorithm for Master Admin Router's TLS. (DCOS-21958)
```

Comment:
I am aware of internal ticket being mentioned in the changelog for the open DC/OS.

If we have a DCOS_OSS JIRA, we will substitute it, otherwise, I am willing to live it for this changelog, as we have already done so for 1.11.2 release. Same for other entries.

Reply:
I've cloned the issue to OSS: https://jira.mesosphere.com/browse/DCOS_OSS-3537, so you can mention it in Open SC/OS release notes.